public-ui/src/app/data-browser/views/survey-view/survey-view.component.html

```diff
@@ -36,12 +36,20 @@ <h2 class="secondary-display">
         </div>
         <div class="stat-container">
           <h2 class="secondary-display">
-            <span>{{ surveyResult.items.length }} </span>
+            <span>{{ survey.questionCount }} </span>
           </h2>
           <p class="info-text">
-            Questions displayed
+            Questions Available
```

Comment:
Display actual questions available count in survey and also matching question count in case the user is searching for something.